src/service/resources-impl.js

```diff
@@ -262,6 +260,13 @@ export class ResourcesDef extends MutatorsAndOwnersDef {
    */
   removeForChildWindow(childWin) {}
 
+  /**
+   * @param {!Resource} resource
+   * @param {boolean} layout
```

Comment:
nit: `layout` => `layoutNeeded`

Reply:
hmm, i think `layoutNeeded` is mis-leading. This boolean only controls the priority of the task.

we should refactor this boolean to `isPreload` (which is negative of the current boolean `layout`)